core/snippets/snippets/classDeclaration.snippet

```diff
@@ -0,0 +1,23 @@
+name: classDeclaration
+insertionScope: class | statement
```

Comment:
I noticed this one's missing a phrase? Is there a way to insert this as-is?

Reply:
That is on purpose for me, but should probably be updated for community.

Reply:
My spoken form to insert a class is `class my class`
https://github.com/AndreasArvidsson/andreas-talon/blob/f9752c7abd547dcad7635955698cee83593f3c33/languages/python/python.py#L119-L120

Reply:
Phrases added

Reply:
> My spoken form to insert a class is `class my class`
> 
> https://github.com/AndreasArvidsson/andreas-talon/blob/f9752c7abd547dcad7635955698cee83593f3c33/languages/python/python.py#L119-L120

Separately, it might be worth discussing if we want move community's language implementations in this direction (lists etc).

Reply:
Sounds good